### ISSUE TITLE
Freya: loki-wallet-cli won't try ipv4 for dual ipv4/ipv6 hostnames (including localhost!)

### ISSUE THREAD

**Issue by jagerman** [loki-project/loki#353]
Because of #352 I disabled IPv6 for my lokid.  I then tried creating a new wallet using `loki-wallet-cli` and I get:

```
Error: wallet failed to connect to daemon: http://localhost:38157. Daemon either is not started or wrong port was passed. Please make sure daemon is running or change the daemon address using the 'set_daemon' command.
```

The same happens if attempting to load a wallet.

If I try connecting to a public node with both ipv4 and ipv6 addresses but that is only listening on ipv4 I get the same error (the hostname here replaced with example.com):

```
Error: wallet failed to connect to daemon: http://example.com:22023. Daemon either is not started or wrong port was passed. Please make sure daemon is running or change the daemon address using the 'set_daemon' command.
```

If I connect with `--daemon-host (IPv4 addr)` the sync works fine.

What's even worse is that I *also* get this connection error even when IPv6 support is enabled in the daemon but I try to connect from a host without IPv6 connectivity.

Given this and #352, perhaps IPv6 support should be pushed back to the next release?

**Comment by KeeJef** [comment]
We have tested on our end and looks like we were able to fix this with #ff0e2f4 will close this issue if its solved on your end to @jagerman

**Comment by jagerman** [comment]
Yup, confirmed fixed in 2.0.1.
